samples/excel/89-preview-apis/chart-trendlines.yaml

```diff
@@ -0,0 +1,200 @@
+                    .forceCreateSheet(context.workbook, "Sample");
+                let expensesTable = sheet.tables.add('A1:E1', true);
+                expensesTable.name = "SalesTable";
+
+                expensesTable.getHeaderRowRange().values = [["Product", "Qtr1", "Qtr2", "Qtr3", "Qtr4"]];
+
+                expensesTable.rows.add(null, [
+                    ["Frames", 5000, 7000, 6544, 4377],
+                    ["Saddles", 400, 323, 276, 651],
+                    ["Brake levers", 12000, 8766, 8456, 9812],
+                    ["Chains", 1550, 1088, 692, 853],
+                    ["Mirrors", 225, 600, 923, 544],
+                    ["Spokes", 6005, 7634, 4589, 8765]
+                ]);
+
+                if (Office.context.requirements.isSetSupported("ExcelApi", 1.7)) {
+                    sheet.getUsedRange().format.autofitColumns();
+                    sheet.getUsedRange().format.autofitRows();
+                }
+
+                createChart(context);
+                sheet.activate();
+
+                await context.sync();
+            });
+        }
+
+        async function createChart(context: Excel.RequestContext) {
+            const sheet = context.workbook.worksheets.getItem("Sample");
+            const salesTable = sheet.tables.getItem("SalesTable");
+
+            const dataRange = salesTable.getDataBodyRange();
+
+            let chart = sheet.charts.add("ColumnClustered", dataRange, Excel.ChartSeriesBy.columns);
+
+            chart.setPosition("A15", "F30");
+            chart.title.text = "Quarterly sales chart";
+            chart.legend.position = "right"
+            chart.legend.format.fill.setSolidColor("white");
+            chart.dataLabels.format.font.size = 15;
+            chart.dataLabels.format.font.color = "black";
+            let points = chart.series.getItemAt(0).points;
+            points.getItemAt(0).format.fill.setSolidColor("pink");
+            points.getItemAt(1).format.fill.setSolidColor("indigo");
+        }
+
+        /** Default helper for invoking an action and handling errors. */
+        async function tryCatch(callback) {
+            try {
+                await callback();
+            }
+            catch (error) {
+                OfficeHelpers.UI.notify(error);
+                OfficeHelpers.Utilities.log(error);
+            }
+        }
+
+
+
+    language: typescript
+template:
+    content: |-
+        <section class="ms-font-m">
+            <p>This sample shows how to add, get, and format trendlines in a chart.</p>
+        </section>
+
+        <section class="setup ms-font-m">
+            <h3>Set up</h3>
+            <button id="setup" class="ms-Button">
+                <span class="ms-Button-label">Add sample data</span>
+            </button>
+        </section>
+
+        <section class="samples ms-font-m">
+            <h3>Try it out</h3>
+            <button id="add-trendline-polynomial" class="ms-Button">
```

Comment:
Now that you're no longer showing a polynomial trendline:
- change name of button to:   add-trendline

Reply:
Done.